default/scripting/species/SP_FULVER.focs.txt

```diff
@@ -42,6 +42,15 @@ Species
         [[LARGE_PLANET]]
         [[NARROW_EP]]
         [[STANDARD_SHIP_SHIELDS]]
+		
+		EffectsGroup
```

Comment:
Weird indenting. Are there tabs used?

Reply:
yeah looks like its 2 tabs, notepad++ just defaulted to that when I started a new line. Its in line with [[STANDARD_SHIP_SHIELDS]] when I look at it in npp

Reply:
should replace tabs with 4 spaces

Reply:
should be fixed below